docs/style.css

```diff
@@ -0,0 +1,386 @@
+/* Full CSS Reset */
+* {
+    all: initial;
+}
+
+/* Re-establish defaults for essential elements */
+*, *::before, *::after {
+    box-sizing: border-box;
+}
+
+/* CSS Variables */
+:root {
+    /* Colors */
+    --color-text-primary: #24292e;
```

Comment:
prefer hsl values. and feel free to value the shades and then use them semantically, e.g.

```css
--white: hsl();


/* later... */
--color-background: var(--white);